src/store/author/reducer.js

```diff
@@ -0,0 +1,16 @@
+import { CREATE_NEW_AUTHOR, SET_AUTHORS_LIST } from './types';
+
+const authorInitialState = [];
+
+export default function authorReducer(state = authorInitialState, action) {
+	switch (action.type) {
+		case SET_AUTHORS_LIST:
+			return [...new Set(action.payload)];
```

Comment:
It seems you can use there return action.payload (in case action.payload will be an array)

Reply:
fixed, thanks